app/cells/datagrid_filter_cell.rb

```diff
@@ -0,0 +1,24 @@
+class DatagridFilterCell < FormCellBase
+  def show
+    case options[:filter].type
+    when :enum
+      cell(
+        :select,
+        nil,
+        select_options
+      ).call(:show)
```

Comment:
I think we should use the external interface (i.e. the `ui_component` helper method) when referencing other components. The implementation may change (quite soon).